bin/file_extension.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import sys
+
+from utils import json_dump, package_validation_dict
+
+
+def file_extension_test(file_path: str) -> str:
+    """
+    1. Reads file extension from provided path
+    2. Checks for valid file extension (.txt is used for now, future version will use ome.tiff and ome.tif)
+
+    Args:
+        file_path (str): Path to the downloaded file
+
+    Returns:
+        str: Result of File extension test
+    """
+    # hard-coded extension check for now, will introduce mapping when we have at least 2 file types to check
```

Comment:
Nit: In vscode, if you did `# TODO` it actually changes the text color which will bring your eyes to it when developing.